src/e2e/java/teammates/e2e/cases/AdminNotificationsPageE2ETest.java

```diff
@@ -0,0 +1,76 @@
+package teammates.e2e.cases;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.NotificationStyle;
+import teammates.common.datatransfer.NotificationTargetUser;
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.TimeHelper;
+import teammates.e2e.pageobjects.AdminNotificationsPage;
+
+/**
+ * SUT: {@link Const.WebPageURIs#ADMIN_NOTIFICATIONS_PAGE}.
+ */
+public class AdminNotificationsPageE2ETest extends BaseE2ETestCase {
+    private NotificationAttributes[] notifications = new NotificationAttributes[2];
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/AdminNotificationsPageE2ETest.json");
+        removeAndRestoreDataBundle(testData);
+
+        notifications[0] = testData.notifications.get("notification1");
+        notifications[1] = testData.notifications.get("notification2");
+    }
+
+    @Test
+    @Override
+    public void testAll() {
+
+        ______TS("verify loaded data");
+
```

Comment:
```suggestion
```
A nitpick: This empty line could be removed.

Reply:
You're right, we should keep this consistent, I've made the fix!